grin/src/stratumserver.rs

```diff
@@ -0,0 +1,518 @@
+// Copyright 2018 The Grin Developers
```

Comment:
Not 100% sure about this but maybe we should have a separate mining crate. We could have the stratum server, the existing miner, and the block building facility in there. @yeastplume what do you think?

Reply:
Perhaps we can extend the `pow` crate that exists at the moment to contain all of this? At the moment, what's actually contained in `pow` is looking a little thin to justify its own crate... however 'pow' is a bit conceptually distinct from 'mining' so I'm not sure.